routes/handlers.go

```diff
@@ -81,3 +81,21 @@ func DropTable(c *fiber.Ctx) error {
 
 	return c.RedirectBack("/")
 }
+
+/*
+	AddColumn takes the table name from the post request
+
+	and calls the AddColumn of the db package.
+*/
+func AddColumn(c *fiber.Ctx) error {
+	table := TableStruct{}
+	column := ColumnStruct{}
+	if err := c.BodyParser(&table); err != nil {
+		return err
+	}
```

Comment:
```suggestion
	}
	if err := c.BodyParser(&column); err != nil {
		return err
	}
```

Everything is working :+1:  you just need to parse the column name and type from the request.